src/assets/translations/en/main.json

```diff
@@ -1046,6 +1047,12 @@
       "sevenDays": "7 days",
       "twentyEightDays": "28 days",
       "custom": "Custom"
+    },
+    "errors": {
+      "nativeSellAssetNotSupported": "Native sell asset not supported",
```

Comment:
Tagging @shapeshift/product here for copy follow-up, IMO, we probably don't want to tell users about "native assets" but be more explicit e.g something like "Cannot use ETH as a limit order sell asset" or similar